code/modules/mob/transform_procs.dm

```diff
@@ -460,8 +460,8 @@
 	. = new_slime
 	qdel(src)
 
-/mob/proc/become_overmind(starting_points = 60)
-	var/mob/camera/blob/B = new /mob/camera/blob(loc, starting_points)
+/mob/proc/become_overmind(starting_points = 60, var/turf/T = loc)

```

Comment:
you don't need `var/turf/` here, because the proc doesn't care about what type it is.
also, please replace T with a more obvious argument name.
also, please don't use vars in a proc's default arguments.